internal/benchmarks/test/fixtures/forc-projects/call-test-contract/src/main.sw

```diff
@@ -0,0 +1,193 @@
+contract;
+
+use std::logging::log;
+use std::context::*;
+use std::call_frames::*;
+use std::registers::context_gas;
+use std::contract_id::ContractId;
+use std::bytes::Bytes;
+
+enum TestB256Enum {
+    Value: b256,
+    Data: b256,
+}
+
+enum TestBoolEnum {
+    Value: bool,
+    Data: bool,
+}
+
+enum TestStringEnum {
+    Value: str[3],
+    Data: str[3],
+}
+
+struct TestStruct {
+    a: bool,
+    b: u64,
+}
+
+struct SumStruct {
+    a: u64,
+    b: u64,
+}
+
+struct SingleParamStruct {
+    a: u64,
+}
+
+abi TestContract {
```

Comment:
@maschad Should we add a contract fn to Mint coins?

Reply:
For sure @Torres-ssf a0597b9d7